src/order/cover.lean

```diff
@@ -177,6 +195,18 @@ lemma wcovby_iff_covby_or_le_and_le : a ⩿ b ↔ a ⋖ b ∨ (a ≤ b ∧ b ≤
 ⟨λ h, or_iff_not_imp_right.mpr $ λ h', h.covby_of_not_le $ λ hba, h' ⟨h.le, hba⟩,
   λ h', h'.elim (λ h, h.wcovby) (λ h, h.1.wcovby_of_le h.2)⟩
 
+lemma covby.imp_left (h : a ⋖ c) (hab : antisymm_rel (≤) a b) : b ⋖ c :=
+⟨hab.2.trans_lt h.lt, λ d hbd hdc, h.2 (hab.1.trans_lt hbd) hdc⟩
+
+lemma covby_congr_left (hab : antisymm_rel (≤) a b) : a ⋖ c ↔ b ⋖ c :=
+⟨λ h, h.imp_left hab, λ h, h.imp_left hab.symm⟩
+
+lemma covby.imp_right (h : c ⋖ a) (hab : antisymm_rel (≤) a b) : c ⋖ b :=
```

Comment:
```suggestion
lemma covby.trans_antisymm_le (h : c ⋖ a) (hab : antisymm_rel (≤) a b) : c ⋖ b :=
```
might be a better name

Reply:
I think it's a better name, but still suboptimal since `antisymm_le` seems like it would spell out some lemma about the antisymmetry of the relation `≤` rather than its antisymmetrization. Maybe we could rename `antisymm_rel` to `indist_rel` in a future PR to alleviate this?